website/snippets/_recommendation-overwriting-manifest.md

```diff
@@ -0,0 +1,8 @@
+To prevent the `manifest.json` from being overwritten before dbt reads it for change detection, update your workflow using one of these methods:
+
+- Move the `manifest.json` to a dedicated folder (for example `state/`) after dbt generates it in the `target/ folder` and, use a saved state, instead of comparing the current state with the just-overwritten version.
```

Comment:
splitting for better readability
```suggestion
- Move the `manifest.json` to a dedicated folder (for example `state/`) after dbt generates it in the `target/ folder`. This makes sure dbt references e a saved state instead of comparing the current state with the just-overwritten version.
```